docs/run.md

```diff
@@ -20,7 +23,10 @@ docker run -p 5150:5150 -v `pwd`/configs:/etc/onos-config-manager -it onosprojec
     -configStore=/etc/onos-config-manager/configStore-sample.json \
     -changeStore=/etc/onos-config-manager/changeStore-sample.json \
     -deviceStore=/etc/onos-config-manager/deviceStore-sample.json \
-    -networkStore=/etc/onos-config-manager/networkStore-sample.json
+    -networkStore=/etc/onos-config-manager/networkStore-sample.json \
+    -caPath tools/test/devicesim/certs/onfca.crt \
```

Comment:
Will this know where do get the files? Did you try it out?

Reply:
No... I wasn't really fixing this... I care that the top part is fixed... docker can be addressed separately.

Reply:
Sorry wasn't trying to be awkward, it's just that I thought it would be more consistent to have an absolute path. I'll try it out here with Docker image

Reply:
No worries... I was just fixing the parts that were clearly broken, but still working on the side with Andrea on the subscribe so did not have time to test the docker bits. I figured more change would come here shortly anyway from @kuujo.